components/IscnRegisterForm.vue

```diff
@@ -1114,38 +1115,48 @@ export default class IscnRegisterForm extends Vue {
   }
 
   async mounted() {
-  if (this.epubMetadata) {
-    this.name = this.epubMetadata.title;
-    this.description = this.extractText(this.epubMetadata.description);
-    this.author.name = this.epubMetadata.author;
-    this.author.authorDescription = 'Author'
-    this.language = this.epubMetadata.language
-    this.tags = this.epubMetadata.tags
-    this.thumbnailUrl = this.formatArweave(this.epubMetadata.thumbnailUrl) as string
-    if (this.author.name) {
-      this.authors.push(this.author)
+    this.uploadStatus = 'loading'
+
+    if (this.epubMetadata) {
+      this.name = this.epubMetadata.title;
+      this.description = this.extractText(this.epubMetadata.description);
+      this.author.name = this.epubMetadata.author;
+      this.author.authorDescription = 'Author'
+      this.language = this.epubMetadata.language
+      this.tags = this.epubMetadata.tags
+      this.thumbnailUrl = this.formatArweave(this.epubMetadata.thumbnailUrl) as string
+      if (this.author.name) {
+        this.authors.push(this.author)
+      }
     }
-  }
-  if (this.address) {
-    const iscnOwner = {
-      name: this.$t('iscn.meta.stakeholders.name.placeholder') as string,
-      wallet: [{
-        content: this.address,
-        id: 1,
-        type: 'like',
-        isOpenOptions: false,
-      }],
-      url: [],
-      likerId: '',
-      authorDescription: 'ISCN owner',
+
+    if (this.address) {
+      try {
+        const { data } = await this.$axios.get(
+          getUserInfoMinByAddress(this.address),
+        )
```

Comment:
Will this throw 404 error, skipping the whole `iscnOwner` block if the address does not have likeId?

Reply:
[🎨 Use default info when likerid not found](https://github.com/likecoin/app-like-co/pull/440/commits/1f52588176c140fb6d65b6939d6182a5ea3228d0)